examples/flutter_infinite_list/lib/posts/bloc/post_state.dart

```diff
@@ -9,14 +9,14 @@ class PostState extends Equatable {
     this.hasReachedMax = false,
   });
 
-  final PostStatus status;
-  final List<Post> posts;
-  final bool hasReachedMax;
+  final PostStatus? status;
+  final List<Post>? posts;
+  final bool? hasReachedMax;
```

Comment:
I don't think any of these should be nullable since they all should have defaults.